### PR TITLE
[refactor](profile) refactor the join profile when its shared hash table

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes
in join node, if it's broadcast_join
and shared hash table, some counter/timer about build hash table is useless,
so we could add those counter/timer in faker profile, and those will not display in web profile.

Issue Number: close #xxx

<--Describe your changes.-->
before

after2

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

File: be/src/exec/exec_node.h [hunk 1]
```diff
@@ -289,6 +290,13 @@ class ExecNode {
     //
     OpentelemetrySpan _span;
 
+    //NOTICE: now add a faker profile, because sometimes the profile record is useless
+    //so we want remove some counters and timers, eg: in join node, if it's broadcast_join
+    //and shared hash table, some counter/timer about build hash table is useless,
+    //so we could add those counter/timer in faker profile, and those will not display in web profile.
+    std::shared_ptr<RuntimeProfile> _faker_runtime_profile =
```
Comment (HappenLee) [inline be/src/exec/exec_node.h:297]: why use shared_ptr ?

**Comment by HappenLee** [comment]
Add picture before and after change ？

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run p0

**Comment by zhangstar333** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Pull request merged**